engine/lib/views.php

```diff
@@ -1342,20 +1373,19 @@ function elgg_view_tagcloud(array $options = array()) {
  *
  * @param \ElggEntity|\ElggAnnotation $item
  * @param array  $vars Additional parameters for the rendering
- *
+ *      'entity_view'     Alternative view to render ElggEntity
+ *      'annotation_view' Alternative view to render ElggAnnotation
+ *      'river_view'      Alternative view to render ElggRiverItem
```

Comment:
For description, how about:

* "View used to render entities (optional)"
* "View used to render annotations (optional)"
* "View used to render river items (optional)"